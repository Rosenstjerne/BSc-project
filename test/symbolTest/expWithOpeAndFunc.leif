var int result;
function int multiply(int x, int y) {
    return x * y;
}

result = (2 + 3) * multiply(4, 5);
print(result); # Should print 100