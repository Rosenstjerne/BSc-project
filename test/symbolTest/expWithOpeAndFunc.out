function null main() {
	var int result;
	
	function int multiply(int x, int y) {
		return (x * y);
		
	}
	
	result = ((2 + 3) * multiply(4, 5));
	print(result);
	
}
